core.js

```diff
@@ -994,6 +994,13 @@ class FileTypeParser {
 			};
 		}
 
+		if (this.check([0x41, 0x43])) {
```

Comment:
You mention `41 43 31 30` could work as well. 
Why don't you use that one? It would decreases the risk on a false positive significant.

Reply:
sure, changed the line to use the longer sequence `41 43 31 30`